bedrock/base/templates/includes/protocol/footer/firefox-new.html

```diff
@@ -0,0 +1,96 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<footer class="mzp-c-footer mzp-t-firefox" id="colophon">
+  <div class="mzp-l-content">
+    <nav class="mzp-c-footer-primary">
+      <div class="mzp-c-footer-sections">
+        <section class="mzp-c-footer-section">
+          <h5 class="mzp-c-footer-heading">
+            Firefox
+          </h5>
+          <ul class="mzp-c-footer-list">
+            <li><a href="{{ url('privacy.notices.firefox') }}" data-link-type="footer" data-link-name="Privacy">{{ _('Privacy') }}</a></li>
+            <li><a href="{{ press_blog_url() }}" data-link-type="footer" data-link-name="Press">{{ _('Press') }}</a>
+            <li><a href="https://mozilla.design/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=design" data-link-type="footer" data-link-name="Design Site">{{ _('Design Site') }}</a>
+          </ul>
+        </section>
+        <section class="mzp-c-footer-section">
+          <h5 class="mzp-c-footer-heading">
+            {{ _('Browsers') }}
+          </h5>
+          <ul class="mzp-c-footer-list">
+            <li><a href="{{ url('firefox.new') }}" data-link-type="footer" data-link-name="Desktop">{{ _('Desktop') }}</a></li>
+            <li><a href="{{ url('firefox.mobile') }}" data-link-type="footer" data-link-name="Mobile">{{ _('Mobile') }}</a></li>
+            <li><a href="https://mixedreality.mozilla.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=reality" data-link-type="footer" data-link-name="Reality">{{ _('Reality') }}</a></li>
+            <li><a href="{{ url('firefox.enterprise.index') }}" data-link-type="footer" data-link-name="Enterprise">{{ _('Enterprise') }}</a></li>
+          </ul>
+        </section>
+        <section class="mzp-c-footer-section">
+          <h5 class="mzp-c-footer-heading">
+            {{ _('Products') }}
+          </h5>
+          <ul class="mzp-c-footer-list">
+            <li><a href="https://lockwise.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=lockwise" data-link-type="footer" data-link-name="Lockwise">{{ _('Lockwise') }}</a></li>
+            <li><a href="https://monitor.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=monitor" data-link-type="footer" data-link-name="Monitor">{{ _('Monitor') }}</a></li>
+            <li><a href="https://send.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer&utm_content=send" data-link-type="footer" data-link-name="Send">{{ _('Send') }}</a></li>
+            <li><a href="#" data-link-type="footer" data-link-name="Browsers">{{ _('Browsers') }}</a></li>
```

Comment:
Link to do (I know you know, just noting em as I find em so we don't miss them).

Reply:
I don't know if we should consider "Browsers" in this context as part of a brand name or just a generic term... I'm leaning toward generic term that should be translated, so let's leave it wrapped.